hack/make/.integration-test-helpers

```diff
@@ -64,8 +64,13 @@ run_test_integration() {
 }
 
 run_test_integration_suites() {
-	local flags="-test.v -test.timeout=${TIMEOUT} $TESTFLAGS"
 	local dirs="$1"
+	local flags="-test.v -test.timeout=${TIMEOUT} $TESTFLAGS "
+	if [ "$dirs" == "integration-cli" ]; then
+		flags="$flags $TESTFLAGS_INTEGRATION_CLI"
```

Comment:
Actually wondering if, instead of introducing two new vars, we could just use `TESTFLAGS` here? They're effectively identical except for the leading `/`.


```bash
	if [ "$dirs" == "integration-cli" ]; then
		flags="$flags -test.run /${TEST_FILTER}"
	else
		flags="$flags -test.run ${TEST_FILTER}"
	fi
```

We probably need to check if `TEST_FILTER` is not empty in that case (I didn't add that to my example)

Reply:
Oh, that's a really great idea. Thanks!